controller/RestQtiItem.php

```diff
@@ -165,7 +165,7 @@ public function importDeferred()
             $taskLog = $this->getServiceManager()->get(TaskLogInterface::SERVICE_ID);
 
             if ($report = $taskLog->getReport($task->getId())) {
-                $result['report'] = $report->toArray();
+                $result['report'] = $this->getReportAsAssociativeArray($report);
```

Comment:
the two format is different, let's not touch it and keep backward compatibility

Reply:
Done :)